perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="lidar_detection_mode" default="camera_lidar_fusion" description="options: `camera_lidar_fusion`, `lidar` or `camera`"/>
+
+  <!-- lidar param -->
+  <arg name="input/pointcloud" />
+  <arg name="input/obstacle_segmentation/pointcloud" default="/perception/obstacle_segmentation/pointcloud"/>
+
+  <!-- camera param -->
+  <arg name="image_raw0" default="/image_raw" description="image raw topic name"/>
+  <arg name="camera_info0" default="/camera_info" description="camera info topic name"/>
+  <arg name="image_raw1" default="/image_raw1"/>
+  <arg name="camera_info1" default="/camera_info1"/>
+  <arg name="image_raw2" default="/image_raw2"/>
+  <arg name="camera_info2" default="/camera_info2"/>
+  <arg name="image_raw3" default="/image_raw3"/>
+  <arg name="camera_info3" default="/camera_info3"/>
+  <arg name="image_raw4" default="/image_raw4"/>
+  <arg name="camera_info4" default="/camera_info4"/>
+  <arg name="image_raw5" default="/image_raw5"/>
+  <arg name="camera_info5" default="/camera_info5"/>
+  <arg name="image_raw6" default="/image_raw6"/>
+  <arg name="camera_info6" default="/camera_info6"/>
+  <arg name="image_raw7" default="/image_raw7"/>
+  <arg name="camera_info7" default="/camera_info7"/>
+  <arg name="image_number" default="1" description="choose image raw number(0-7)"/>
+  <arg name="lidar_detection_model" default="centerpoint" description="options: `centerpoint`, `apollo`"/>
+  <arg name="use_pointcloud_map" default="true" description="use pointcloud map in detection"/>
+
+  <!-- camera lidar fusion based detection-->
+  <group if="$(eval '&quot;$(var lidar_detection_mode)&quot;==&quot;camera_lidar_fusion&quot;')">
```

Comment:
https://github.com/tier4/autoware_launch/pull/387/files#r914435704
Same comment

Reply:
After that, pls add namespace `camera_lidar_fusion`